mk/aosp_optee.mk

```diff
@@ -86,15 +86,24 @@ $(LOCAL_PREBUILT_MODULE_FILE): $(TA_TMP_FILE)
 	@mkdir -p $(dir $@)
 	cp -uvf $< $@
 
+TA_TMP_FILE_DEPS :=
+ifneq ($(local_dependency_ta), )
```

Comment:
I find the name slightly obscure, and it does not show that several dependencies may be given. How about `local_module_deps` instead? Usage like so:
```
local_module := <some-uuid>.ta
local_module_deps := libfoo.so libbar.so
```

Reply:
ok